samples/Calling/src/app/ConfigurationScreen.tsx

```diff
@@ -77,7 +77,7 @@ export const ConfigurationScreen = (props: ConfigurationScreenProps): JSX.Elemen
               startCallHandler();
             }
           }}
-          isDisabled={!displayName || emptyWarning || nameTooLongWarning}
+          isDisabled={!displayName || emptyWarning || nameTooLongWarning || !microphonePermissionGranted}
```

Comment:
Same here for this one

Reply:
Going to leave it as is due to David's requirements I posted above. But if it becomes an issue we can revisit it.